google-codelab-index/google_codelab_index_cards.js

```diff
@@ -176,6 +190,33 @@ class Cards extends HTMLElement {
         .trim().split(',').shift();
   }
 
+  /**
+   * Trims whitespace and converts to lower case.
+   * @param {string|undefined} v
+   * @return {string}
+   * @private
+   */
+  normalizeValue_(v) {
+    return (v || '').trim().toLowerCase()
+        .replace('\n', '')
+        .replace(/\s+/g, ' ');
+  }
+
+  /**
+   * @param {!Array<string>} strings 
+   * @return {!Array<string>}
+   * @private
+   */
+  /*cleanStrings_(strings) {
```

Comment:
No need for this anymore, I guess?

Reply:
Nar I just haven't gotten round to that bit yet :)